### ISSUE TITLE
In the last email from info@microbit.org did you mean 7 bit ascii and not UTF8?

### ISSUE THREAD

**Issue by rhubarbdog** [microsoft/pxt-microbit#2235]
**Describe the bug**
I'm into thermometers. I like the ° symbol but when i run the attached blocks my serial terminal prints �
in the last email from info@microbit.org there's a link to <a href="https://support.microbit.org/support/solutions/articles/19000098698">this page</a>
i quote "UTF8 character support in strings"

if i run this MicroPython program
```
for i in range(128,256,8):
    for j in range(8):
        print(i+j,chr(i+j),end='\t')
    print("")
```
it yeilds
```
128     129     130     131     132 
                                        133 
    134         135                             142     143 
144     145    146    147    148    149 	150 	151 	
152 	153 	154 	155 	156     157     158     159 
160     161 ¡   162 ¢   163 £   164 ¤   165 ¥   166 ¦   167 §   
168 ¨   169 ©   170 ª   171 «   172 ¬   173 ­   174 ®   175 ¯   
176 °   177 ±   178 ²   179 ³   180 ´   181 µ   182 ¶   183 ·   
184 ¸   185 ¹   186 º   187 »   188 ¼   189 ½   190 ¾   191 ¿   
192 À   193 Á   194 Â   195 Ã   196 Ä   197 Å   198 Æ   199 Ç   
200 È   201 É   202 Ê   203 Ë   204 Ì   205 Í   206 Î   207 Ï   
208 Ð   209 Ñ   210 Ò   211 Ó   212 Ô   213 Õ   214 Ö   215 ×   
216 Ø   217 Ù   218 Ú   219 Û   220 Ü   221 Ý   222 Þ   223 ß   
224 à   225 á   226 â   227 ã   228 ä   229 å   230 æ   231 ç   
232 è   233 é   234 ê   235 ë   236 ì   237 í   238 î   239 ï   
240 ð   241 ñ   242 ò   243 ó   244 ô   245 õ   246 ö   247 ÷   
248 ø   249 ù   250 ú   251 û   252 ü   253 ý   254 þ   255 ÿ 
```
These are all single byte ascii characters as per https://github.com/microsoft/pxt-microbit/issues/2186
**To Reproduce**
1. download, flash and execute these blocks https://makecode.microbit.org/_878MgKJHX93D
2. start a serial terminal e.g. `screen /dev/ttyACM0 112500`
3. press button B and the � is printed

**Expected behavior**
I expect the ° symbol to be printed when button B is pressed

**Desktop (please complete the following information):**
 - OS: Linux mint 18.2
 - Browser mozilla Firefox
 - Version 67.0

**Comment by martinwork** [comment]
I don't know what the missing text under "UTF8 character support in strings" would have been, but have discovered the following...

micropython is sending UTF8, which is interpreted by the Linux terminal.  The degree symbol is sent as four bytes: C3, 82, C2, B0. In TeraTerm, I had to change Setup/General/Language to UTF8 and change the font to Courier to see a table similar to the above.

The blocks example shows the degree symbol in the MakeCode console on my Windows computer. It is sending a simple byte 176 for the degree symbol. In TeraTerm, I had to change the font to Courier.

This page suggests using ttyUSBx rather than ttyACMx:
https://support.microbit.org/support/solutions/articles/19000022103-outputing-serial-data-from-the-micro-bit-to-a-computer

Single byte character codes in the range 128-255 have many interpretations:
https://en.wikipedia.org/wiki/ASCII
https://en.wikipedia.org/wiki/Extended_ASCII
